src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -410,7 +431,7 @@ def send_comm_message(self, msg_type: str, content: dict) -> None:
         self._comm.send(msg)
 
     def send_error_message(
-        self, err_type: str, req: JobRequest, content: dict = None
+        self, err_type: str, req: JobRequest, content: dict = None, job_id_key: str = "job_id"
```

Comment:
Allow `send_error_message` to send error message with `job_id_list` instead of `job_id` for cases of `job_comm_error`